tests/_core/geodistance_query.yaml

```diff
@@ -0,0 +1,60 @@
+$schema: ../../json_schemas/test_story.schema.yaml
+
+description: Test search endpoint with geo_distance query.
+prologues:
+  - path: /test_index
```

Comment:
Nit: let's rename `test_index` to something meaningful like `games` or `movies`. For Geo maybe we can use `map`?